app/javascript/app/utils/index.js

```diff
@@ -7,4 +7,5 @@ const { LoginGov } = window;
 
 LoginGov.autoLogout = autoLogout;
 LoginGov.countdownTimer = countdownTimer;
+LoginGov.srCountdownTimer = countdownTimer;
```

Comment:
It looks like the timing of the behavior is synced with the normal countdown timer, should we just have the one?
```suggestion
LoginGov.countdownTimer = countdownTimer;
```